docs/_articles/en/user-guide/detect-conflicts.md

```diff
@@ -22,7 +22,7 @@ Because the conflict detection feature is in beta, you must enable the feature:
 
 You can also enter conflict detection in the search box to find the feature and then enable it.
 
-In this beta release, we have enabled conflict detection for the **SFDX: Retrieve Source in Manifest from Org** and **SFDX: Deploy Source in Manifest to Org** commands only. All manifest files that exists both in the org and in the local environment are checked for conflicts; files that don’t exist in both are not checked.
+Once enabled, conflict detection is available on all commands that deploy and retrieve source.
```

Comment:
@sbudhirajadoc With this PR, conflict detection will change to use timestamps and will be available for the 'Retrieve source from source path' and soon the 'Deploy source from source path' commands. So at the minimum we need a doc change like this, and we may want to describe the new mechanism as well.